### PR TITLE
fix orderbook subscription bug

### PR DESCRIPTION
**Pull request by shekenahglory**

### CODE REVIEW

**Comment by shekenahglory** [comment]
same fix and tests as on the develop branch

File: src/orderbook.js [hunk 1]
```diff
@@ -326,10 +326,11 @@ OrderBook.prototype.requestOffers = function(callback = function() {},
   internal = false) {
   const self = this;
 
-  if (!this._remote.isConnected()) {
+  if (!this._remote.isConnected() && !internal) {
     // do not make request if not online.
     // that requests will be queued and
     // eventually all of them will fire back
+    callback(new Error('remote is offline'));
```
Comment (clark800) [inline src/orderbook.js]: RippleError?

Comment (shekenahglory) [inline src/orderbook.js]: Thought that was a RippleAPI addition - will fix

**Comment by shekenahglory** [comment]
updated

**Comment by clark800** [comment]
LGTM

**Pull request merged**